### DISCUSSION TITLE
Use of Paste recursive, updating references.

### DISCUSSION THREAD

**Discussion by Jeevraj123** [pimcore/pimcore#17464] [Q&A / Support] [answered]
Actually I need some context around copy and paste recursive, updating references right click options on pimcore.

1.In which scenario this paste type is applicable with any example.
2.When this paste option is useful.

**Comment by markus-moser** [comment, marked as answer]
* Recursive means that you do not only copy the current element but also all children of the current element (recursively through all sub folder levels of the current element)
* Updating references means that it changes all relations within the copied items to the newly created elements. So e.g. if you are coping a data object folder which contains products and brands where the products are linked to the brands the copied products then link to the copied brands instead of the original ones.

**Reply by Jeevraj123** [reply]
Thank you @markus-moser This will be indeed helpful.
